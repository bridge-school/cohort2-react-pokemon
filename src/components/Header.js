import React from 'react';

export const Header = ({ title }) => <header>{ title }</header>;
