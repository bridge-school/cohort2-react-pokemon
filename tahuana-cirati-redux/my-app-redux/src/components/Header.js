import React, { Component } from 'react';
import { connect } from 'react-redux';

class Header extends Component {
    render() {
        return (<div className="App-header">{this.props.header}</div>)
    }
}

const mapStateToProps = state => ({
    header: state.header
})
      
export default connect(mapStateToProps)(Header);
  