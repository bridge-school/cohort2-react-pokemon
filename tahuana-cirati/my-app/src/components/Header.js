import React from 'react';

const Header = ({data}) => (
    <div className="App-header">
        {data}
    </div>
)

export default Header;