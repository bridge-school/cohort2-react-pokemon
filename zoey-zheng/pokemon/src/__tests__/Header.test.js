import React from 'react';
import { Header } from '../Header.js';
import renderer from 'react-test-renderer';

describe('header component tests', () => {
  it('renders header component', () => {
    const header = renderer.create(<Header />);
    expect(header.toJSON()).toMatchSnapshot();
  });
});
