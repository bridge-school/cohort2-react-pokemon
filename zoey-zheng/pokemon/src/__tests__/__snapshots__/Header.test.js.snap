// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header component tests renders header component 1`] = `
<h1>
   
   
</h1>
`;
